x-pack/plugins/serverless_search/server/routes/indices_routes.ts

```diff
@@ -72,6 +74,45 @@ export const registerIndicesRoutes = ({ router, security }: RouteDependencies) =
       });
     }
   );
+
+  router.post(
+    {
+      path: '/internal/serverless_search/indices/{index_name}/search',
+      validate: {
+        body: schema.object({
+          searchQuery: schema.string({
+            defaultValue: '',
+          }),
+        }),
+        params: schema.object({
+          index_name: schema.string(),
+        }),
+        query: schema.object({
+          page: schema.number({ defaultValue: 0, min: 0 }),
+          size: schema.number({
+            defaultValue: DEFAULT_DOCS_PER_PAGE,
+            min: 0,
+          }),
+        }),
+      },
+    },
+    async (context, request, response) => {
+      const client = (await context.core).elasticsearch.client.asCurrentUser;
+      const indexName = decodeURIComponent(request.params.index_name);
+      const searchQuery = request.body.searchQuery;
+      const { page = 0, size = DEFAULT_DOCS_PER_PAGE } = request.query;
+      const from = page * size;
+
+      const searchResults = await fetchSearchResults(client, indexName, searchQuery, from, size);
+      // console.log('searchResults', from, size);
```

Comment:
left over

Reply:
Good catch, let me see if I can find any more of these